### PR TITLE
Update android-dedicated-devices-fully-managed-enroll.md

### PR DESCRIPTION
**Pull request by shanthithillairajah**
Updating article with more accurate information about zero-touch configurations

### CODE REVIEW

File: memdocs/intune/enrollment/android-dedicated-devices-fully-managed-enroll.md [hunk 1]
```diff
@@ -131,20 +131,20 @@ Link a zero-touch account with your Microsoft Intune account. Upon linking the a
...
+6. A default configuration is created. A screen appears with basic information about the new configuration. Intune will automatically apply the default configuration to any zero-touch enabled device that's without an existing configuration. Select **Next** to continue.    
 
 > [!TIP]
-> The token used for the default configuration is for a fully managed device. If you want to create a zero-touch configuration for a corporate-owned work profile device or a dedicated device, see [Create configuration in zero-touch enrollment portal](android-dedicated-devices-fully-managed-enroll.md#create-configuration-in-zero-touch-enrollment-portal) (in this article). 
+> The token used for the default configuration is for a fully managed device. If you want to create a new zero-touch configuration or a zero-touch configuration for a corporate-owned work profile device or a dedicated device, click "View devices in the zero-touch portal" from the iframe to use the zero-touch portal. See [Create configuration in zero-touch enrollment portal](android-dedicated-devices-fully-managed-enroll.md#create-configuration-in-zero-touch-enrollment-portal) (in this article).
```
Comment (JohanFreelancer9) [inline memdocs/intune/enrollment/android-dedicated-devices-fully-managed-enroll.md:137]:
```suggestion
> The token used for the default configuration is for a fully managed device. If you want to create a new zero-touch configuration or a zero-touch configuration for a corporate-owned work profile device or a dedicated device, select **View devices in the zero-touch portal** from the iframe to use the zero-touch portal. See [Create configuration in zero-touch enrollment portal](android-dedicated-devices-fully-managed-enroll.md#create-configuration-in-zero-touch-enrollment-portal) (in this article). 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @shanthithillairajah, kindly commit the minor suggested change. Many thanks.

**Comment by lenewsad** [comment]
#sign-off

**Comment by lenewsad** [comment]
#hold-off

**Comment by lenewsad** [comment]
@JohanFreelancer9 @shanthithillairajah Opening PR and submitting changes in private repo. Closing this PR.

**Pull request closed without merging**